tests/e2e/support/objects/app-files/spaces/index.ts

```diff
@@ -59,13 +59,9 @@ export class Spaces {
     await po.removeAccessSpaceMembers({ ...args, page: this.#page })
   }
 
-  getSpaceID({ key }: { key: string }): string {
-    const { id } = this.#spacesEnvironment.getSpace({ key })
-    return id
-  }
-
-  async expectThatSpacesIdNotExist(spaceID: string): Promise<void> {
-    await spaceWithSpaceIDNotExist({ spaceID, page: this.#page })
+  async expectThatSpacesIdNotExist(space: string): Promise<void> {
+    const { id } = this.#spacesEnvironment.getSpace({ key: space })
+    await spaceWithSpaceIDNotExist({ spaceID: id, page: this.#page })
```

Comment:
you can utilize `getSpaceID`
```suggestion
  async expectThatSpacesIdNotExist(space: string): Promise<void> {
    const spaceID = this.getSpaceID({ key: space })
    await spaceWithSpaceIDNotExist({ spaceID, page: this.#page })
```

Reply:
sorry but why add a function with only one line of code it is not even used by other functions?

Reply:
It can be helpful to have it there. For example, in this code. Using `getSpaceID`, we don't have to remember everytime which property to destruct from `this.#spacesEnvironment.getSpace({ key: space })` to get the space id; property `id` in this case

Reply:
we have tried to implement a similar way in other places.

Reply:
sorry to say but this file contain `const { id } = this.#spacesEnvironment.getSpace({ key: space }` in all places

Reply:
good to have them replaced with `getSpaceID` but not in this PR. Can done in the next refactoring PR